drivers/sensor/bh1750/Kconfig

```diff
@@ -0,0 +1,36 @@
+# BH1750 ambient light sensor configuration options
+
+# Copyright (c) 2022 morsisko
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BH1750
+	bool "BH1750 ambient light sensor"
+	depends on I2C
+	help
+	  Enable driver for BH1750 ambient light sensor.
+
+if BH1750
+	  
+choice
+	prompt "BH1750 resolution mode"
+	default BH1750_HIGHT_RESOLUTION_MODE
+	help
+	  Select resolution mode. The higher the resolution, the more time it
+	  takes to complete one measurement.
+config BH1750_LOW_RESOLUTION_MODE
+	bool "low resolution"
+config BH1750_HIGHT_RESOLUTION_MODE
+	bool "high resolution"
+config BH1750_HIGHT_RESOLUTION_2_MODE
+	bool "high resolution 2"
+endchoice
+
+config BH1750_MEASUREMENT_TIME
+	int "Value of internal measurement time register"
+	range 31 254
+	default 69
+	help
+	  Specify the measurement time. The greater the value, the more time it
+	  takes to complete one measurement and the sensor is more sensitive.

```

Comment:
please move to Devicetree

Reply:
Do you mean I should delete the Kconfig file, and instead grab those settings from bh1750 device tree node?